lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -82,6 +87,13 @@ class CreateCommand < HammerCLIForeman::CreateCommand
       success_message _("Compute resource created.")
       failure_message _("Could not create the compute resource")
 
+      def request_params
+        params = super
+        if options["option_public_key_path"]
+          params['compute_resource']['public_key'] = options["option_public_key_path"]
+        end
+        params
+      end
       build_options
```

Comment:
In the command help I can see:
```
 --public-key PUBLIC_KEY                     For oVirt only
 --public_key_path PUBLIC_KEY_PATH           Path to a file that contains oVirt public key (For oVirt only)
```
This is probably unwanted. Use `build_options :without => :public_key`